dt-users/template-user-management.php

```diff
@@ -270,7 +270,7 @@
                                                 <span class="typeahead__query">
                                                     <input class="js-typeahead-location_grid input-height"
                                                            name="location_grid[query]"
-                                                           placeholder="<?php esc_html_e( "Search Locations", 'disciple_tools' ) ?>"
+                                                           placeholder="<?php esc_html_x( "Search Locations", 'input field placeholder', 'disciple_tools' ) ?>"
```

Comment:
Change not needed.

Reply:
Was done to collect this one string to be used in the previously translated strings - so will not trigger a new translation.

Reply:
I suggest we remove the context from all of them instead since both strings are translated.

Reply:
Removed the context of both.